docs/integrations/builtin/core-nodes/n8n-nodes-base.extractfromfile.md

```diff
@@ -8,44 +8,72 @@ priority: high
 
 # Extract From File
 
-Use the Extract From File node to get data from a binary format and convert it to JSON.
+A common pattern in n8n workflows is to receive a file, either from and [HTTP Request node][] (for files you are fetching from a website), a [Webhook Node][] (for files which are sent to your workflow from elsewhere) or from a local source. Data obtained in this way is usually in a binary format, for example a spreadsheet or PDF.
```

Comment:
```suggestion
A common pattern in n8n workflows is to receive a file, either from and [HTTP Request node][] (for files you are fetching from a website), a [Webhook Node][] (for files which are sent to your workflow from elsewhere), or from a local source. Data obtained in this way is often in a binary format, for example a spreadsheet or PDF.
```
Link style tripped me up for a second!

Reply:
Oh yes, I reverted to my old ways. I do very much like how it makes the text easier to parse though